FParsecCS/Text.cs

```diff
@@ -204,7 +204,11 @@ public static class Text {
 
 /// <summary>Returns the given string with all occurrences of "\r\n" and "\r" replaced
 /// by "\n". If the argument is null, null is returned.</summary>
+#if NET
+public static string NormalizeNewlines(string str) => str?.ReplaceLineEndings("\n");
```

Comment:
That's a behaviour change, since it replaces all Unicode newlines, not just the three FParsec normally handles. Is that intentional?

Reply:
I was misled by the documentation above. Thanks for pointing out. Reverted.